recipes/nmos-cpp/all/conanfile.py

```diff
@@ -40,8 +40,15 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+        # Simple change as github doesn't allow review comments on unchanged lines that are outside the +/- 3 lines of the changes.
+        # To be removed in the review process.
         if self.settings.os == "Macos":
```

Comment:
```suggestion
        if self.settings.os == "Macos":
```
Comment just temporary to get the possibility of doing comments in this review tool here which didn't allow it despite unfolding unchanged lines.